test/shared_utilities/time_varying_inputs.jl

```diff
@@ -102,3 +112,106 @@ end
         end
     end
 end
+
+@testset "InteprolatingTimeVaryingInput2D" begin
+    PATH = ClimaLand.Bucket.cesm2_albedo_dataset_path()
+    for FT in (Float32, Float64)
+        target_space =
+            ClimaLand.Domains.SphericalShell(;
+                radius = FT(6731e3),
+                depth = FT(1.0),
+                nelements = (4, 4),
+                npolynomial = 4,
+            ).space.surface
+        data_handler = DataHandling.DataHandler(
+            PATH,
+            "sw_alb",
+            target_space,
+            reference_date = Dates.DateTime(2000, 1, 1),
+            t_start = 0.0,
+        )
+
+        input_nearest = TimeVaryingInputs.TimeVaryingInput(
+            data_handler,
+            method = TimeVaryingInputs.NearestNeighbor(),
+        )
+
+        @test 0.0 in input_nearest
+        @test !(1e23 in input_nearest)
+
+        dest = Fields.zeros(target_space)
+
+        available_times = DataHandling.available_times(data_handler)
+
+        # We are testing NearestNeighbor, so we can just have to check if the fields agree
+
+        # Left nearest point
+        target_time = available_times[10] + 1
+        TimeVaryingInputs.evaluate!(dest, input_nearest, target_time)
+
+        # We use isequal to handle NaNs
+        @test isequal(
+            Array(parent(dest)),
+            Array(
+                parent(
+                    DataHandling.regridded_snapshot(
+                        data_handler,
+                        available_times[10],
+                    ),
+                ),
+            ),
+        )
+
+        # Right nearest point
+        target_time = available_times[9] - 1
+        TimeVaryingInputs.evaluate!(dest, input_nearest, target_time)
+
+        @test isequal(
+            Array(parent(dest)),
```

Comment:
why do we need to use Array here but not in the "on node" case?

Reply:
I added the Array there too